docs/reference/commandline/cli.md

```diff
@@ -378,6 +378,37 @@ list of root Certificate Authorities.
 
 ## Examples
 
+### <a name="host"></a> Specify daemon host (-H, --host)
+
+You can use the `-H`, `--host` flag to specify one or more sockets to use when
+you invoke a `docker` command. You can use the following protocols:
+
+- `unix://`
+- `tcp://`
+- `ssh://`
+
+The following example shows how to invoke `docker ps` over TCP, to a remote
+daemon with IP address `174.17.0.1`, listening on port `3131`:
+
+```console
+$ docker -H tcp://174.17.0.1:3131 ps
```

Comment:
perhaps use `:2376` here to encourage using the secure connection.

We have 3 ports registered; https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=docker

- `2375`: tcp, non-TLS (insecure)
- `2376`: tcp (TLS)
- `2377`: used for swarm (secure)

So only the first 2 are relevant for this one, but perhaps we need to mention them somewhere

Reply:
I added a small note about it here, but we might want to add a similar description for this somewhere else too. Let's take it in a followup. LMK if the note I added seems OK to you.